Device/DeviceModel/schema.json

```diff
@@ -0,0 +1,165 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "id": "https://fiware.github.io/dataModels/Device/DeviceModel/schema.json",
+  "title": "GSMA / FIWARE - Device Model schema",
+  "description": "This entity captures the static properties of a Device. ",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/GSMA-Commons"
+    },
+    {
+      "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/PhysicalObject-Commons"
+    },
+    {
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "DeviceModel"
+          ],
+          "description": "NGSI Entity type"
+        },
+        "category": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "enum": [
+              "sensor",
+              "actuator",
+              "meter",
+              "HVAC",
+              "network",
+              "multimedia"
+            ]
+          }
+        },
+        "deviceClass": {
+          "type": "string",
+          "enum": [
+            "C0",
+            "C1",
+            "C2"
+          ]
+        },
+        "controlledProperty": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "enum": [
+              "temperature",
+              "humidity",
+              "light",
+              "motion",
+              "fillingLevel",
+              "occupancy",
+              "power",
+              "pressure",
+              "smoke",
+              "energy",
+              "airPollution",
+              "noiseLevel",
+              "weatherConditions",
+              "precipitation",
+              "windSpeed",
+              "windDirection",
+              "barometricPressure",
+              "solarRadiation",
+              "depth",
+              "pH",
+              "conductivity",
+              "conductance",
+              "tss",
+              "tds",
+              "turbidity",
+              "salinity",
+              "orp",
+              "cdom",
+              "waterPollution",
+              "location",
+              "speed",
+              "heading",
+              "weight",
+              "waterConsumption",
+              "gasComsumption",
+              "electricityConsumption"
+            ]
+          }
+        },
+        "function": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "enum": [
+              "levelControl",
+              "sensing",
+              "onOff",
+              "openClose",
+              "metering",
+              "eventNotification"
+            ]
+          }
+        },
+        "supportedProtocol": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "enum": [
```

Comment:
instead of repeating these definitions (which are also at device level) I think we need to create one schema for definitions and reuse it as we did in other cases

Reply:
k